db.js

```diff
@@ -541,3 +541,13 @@ const asanas = [
 ];
 
 export default asanas;
```

Comment:
I think you don't need to export your dummy array, since you wrote service functions below, which have access to the array.

Reply:
I actually use it in the index.js :)